cosmos-gui/README.md

```diff
@@ -181,6 +181,7 @@ cosmos-gui is configured through `conf/cosmos-gui.json`. There you will find a J
     * **user**: a valid user in the MySQL server with permissions to insert into the `cosmos_user` table.
     * **password**: password for the above user in MySQL.
     * **database**: must be `cosmos`.
+* **users_blacklist**: 
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/1df5993ff7b53ba91a11436d570a9b8b9546d1ce